src/components/admin/article/ArticleController.jsx

```diff
@@ -342,23 +357,38 @@ class ArticleController extends FalcorController {
           onRequestClose={this.handleDialogClose}
         >
           {this.state.saving ? <LoadingOverlay /> : null}
-          <TitleField
-            title={this.state.title}
+          <ShortRequiredTextField
+            floatingLabelText="Title"
+            value={this.state.title}
             onUpdate={this.updateTitle}
             disabled={this.state.saving}
           />
+          <ShortRequiredTextField
+            floatingLabelText={`Slug${
+              !article.published_at
+                ? ''
+                : ' - You cannot edit the slug of a published article. Unpublish this article to edit the slug.'
```

Comment:
@Helw150 I'm actually completely fine with it being in label text, I simply meant the full stop at the end of the string

Reply:
Just the final one right?

Reply:
Oh yeah just the final one